apiaudio/api_request.py

```diff
@@ -141,6 +145,16 @@ def _expanded_raise_for_status(self, res):
         @return: None
         """
         try:
+            if res.headers.get("Warning") and apiaudio.log_warnings:
+                for warn in re.findall(r'\"(.*?)\"', res.headers["Warning"]):  # get all messages in between ""
+                    if apiaudio.sdk_version in warn:
```

Comment:
i did not understand this at all, why would `0.1.2` be in warning messages

Reply:
If the warning is concerning the version being non-latest, the current version will be mentioned. I'm changing this as discussed in https://github.com/aflorithmic/api-utils/pull/12 this kind of warning will be signified by a separate header from tracing